src/harness/harnessUtils.ts

```diff
@@ -206,15 +206,10 @@ export function sourceFileToJSON(file: ts.Node): string {
                     break;
 
                 case "hasExtendedUnicodeEscape":
-                    if ((n as any).hasExtendedUnicodeEscape) {
+                    if ((n as ts.Identifier).hasExtendedUnicodeEscape) {

```

Comment:
String literals also have a `hasExtendedUnicodeEscape` property. If you're going to cast, then we should probably reflect that here in case we need to find references later.